categories.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+require('vendor/autoload.php');
+use FurnitureStoreAPI\Categories\CategoryHydrator as CategoryHydrator;
+use FurnitureStoreAPI\Response\Response as Response;
+
+header('Content-Type: application/json; charset=utf-8');
+header("Access-Control-Allow-Origin: *");
```

Comment:
These two lines are going to be repeated in other files too, so please make a `HeaderService` with a static method that sets these headers.